src/app/page.tsx

```diff
@@ -1,7 +1,24 @@
+import Text from "../components/Text";
+
 export default function Home() {
   return (
     <main>
       <p>Welcome to the home page!</p>
+      <Text text="This is plain text." />
+      <Text text="This is italic text." italic/>
+      <Text text="This is bold text." bold/>
+      <Text text="This is underlined text." underline/>
+      <Text text="This text is bold, italic, and underlined." bold italic underline />
```

Comment:
IMHO Component props should appear before the actual text...

I can defer to @RaisaBh for this one in terms of code-practice